src/QtDBMigration.cpp

```diff
@@ -32,9 +32,9 @@ class QtDBMigrationPrivate
     QtDBMigrationPrivate(QtDBMigration *q);
     ~QtDBMigrationPrivate();
 
-    int version();
```

Comment:
Logic in this file needs to stay intact!

Reply:
Previous logic restored back and added the folder config.